app/services/metrics_api/v3/endpoints/root.rb

```diff
@@ -12,14 +12,16 @@ class Root
         METRICS_TO_MODEL = {
           'record' => SupplejackApi::FacetedMetrics,
           'view' => SupplejackApi::UsageMetrics,
-          'view_new' => SupplejackApi::CollectionMetric
+          'view_new' => SupplejackApi::CollectionMetric,
+          'top_records' => SupplejackApi::TopCollectionMetric
         }.freeze
         # Mapping of metrics to the field on it's respective model that contains
         # the value to filter against
         METRIC_TO_MODEL_KEY = {
           'record' => :name,
           'view' => :record_field_value,
-          'view_new' => :display_collection
+          'view_new' => :display_collection,
```

Comment:
what does the `_new` mean?

Reply:
This is the new 'view' that will eventually replace the current 'view' once things are switched over, I think Gus built this a few months ago.